tests/lib/rules/no-extra-parens.js

```diff
@@ -922,6 +926,19 @@ ruleTester.run("no-extra-parens", rule, {
             options: ["all", { ignoreJSX: "none" }]
         }),
 
+        // ["all", { enforceForArrowConditionals: false }]
+        {
+            code: "var a = (b) => (1 ? 2 : 3)",
+            parserOptions: { ecmaVersion: 6 },
+            options: ["all", { enforceForArrowConditionals: false }],
+            errors: [
+                {
+                    message: "Gratuitous parentheses around expression."
+                }
+            ],
+            output: "var a = (b) => 1 ? 2 : 3"
+        },
```

Comment:
Can you add a test for `() => ((1 ? 2 : 3))`? I think an error should be reported because there is no reason to use multiple sets of parentheses there.